syntax = "proto3";

import "alice/boltalka/generative/service/proto/common.proto";

package NGenerativeBoltalka.Proto;

message TEmbeddingRequest {
    repeated string Context = 1;
    TPtuneEmbeddings PtuneEmbeddings = 2;
}
